web/frontend/src/pages/election/Show.tsx

```diff
@@ -124,10 +148,14 @@ const ElectionShow: FC = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [roster]);
 
+  // Keep the "DKGLoading" state according to "nodeLoading". This state tells if
+  // one of the element on the map is true.
   useEffect(() => {
     if (nodeLoading !== null) {
       if (!Array.from(nodeLoading.values()).includes(true)) {
         setDKGLoading(false);
+      } else {
+        setDKGLoading(true);
       }
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
in general exhaustive deps are a very good thing, shouldn't these issues be addressed ?
ignoring them has cost me a lot of time bug fixing, in the past :-/

Reply:
We actually tried a couple of times to include all the dependencies with @badrlarhdir during the semester. But for some reasons, the useEffects were not triggered at the correct/expected moment when doing so... 🤔